addons/website/views/snippets/s_image_text_overlap.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_text_overlap" name="Image - Text Overlap">
+    <section class="s_image_text_overlap pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="9">
+                <div class="o_grid_item g-col-lg-6 g-height-7 col-lg-5 o_cc o_cc5 align-content-center pt16 pb16" style="grid-area: 2 / 7 / 9 / 13; --grid-item-padding-y: 48px; --grid-item-padding-x: 48px; z-index: 2; border-radius: var(--border-radius)">
```

Comment:
`pt16 pb16` -> why? The padding is already set by `--grid-item-padding-[x-y]` (other block as well)